src/ucs/sys/event_set.h

```diff
@@ -46,10 +47,11 @@ extern const unsigned ucs_sys_event_set_max_wait_events;
  * Allocate ucs_sys_event_set_t structure.
  *
  * @param [out] event_set_p  Event set pointer to initialize.
+ * @param [in]  user_fd      User defined event poll file descriptor.
```

Comment:
I want to know that `-1` description in `user_fd` param.

Reply:
done